test/isolated/config.test.js

```diff
@@ -108,4 +108,19 @@ describe('config', () => {
     )
     expect(config.__test__ext).toBe('js')
   })
+
+  it('Should log a warning when no exported configuration found', () => {
```

Comment:
It'd be better to copy `test/integration/config` to `test/integration/config-empty` and check the actual CLI output. That way we ensure it works correctly across any codebase changes.

Reply:
Will do!

Reply:
Lmk if you need pointers to how to check the CLI output, we do it in quite a few places in the test suite.

Reply:
I took an example of `test/integration/config-experimental-warning`, hope it was something you were thinking about?